xmtp/src/storage/encrypted_store/mod.rs

```diff
@@ -132,9 +149,24 @@ impl EncryptedMessageStore {
     }
 }
 
+impl KeyStore for EncryptedMessageStore {
+    fn get_account(&mut self) -> Result<Option<Account>, StorageError> {
+        let mut account_list: Vec<Account> = self.fetch()?;
```

Comment:
Would be helpful to throw an error if this list is ever longer than a single item.

Reply:
I was planning for a future where KeyRotation is implemented and there may be more than one historical account. 

Enforcing at the Schema level would mean throwing away that data.  -- I'm not particularly opposed to that path -- just haven't given it much thought.

Reply:
@neekolas is an error preferred over "Warning + Only return the oldest"?

My fear about the an error is that developers will need a way to resolve the issue, or there install will be bricked

Reply:
That's a fair concern. I'm alright with "warning + return the newest"